src/router/constants/guestRoutes.ts

```diff
@@ -1,5 +1,5 @@
 export const guestRoutes = {
-  home: { route: '/', path: '/' },
+  home: { route: '/', path: '/#welcome' },
```

Comment:
not here, home path should be clear

Reply:
> not here, home path should be clear

cleared home path and created separate route for this